src/controllers/basket.controller.ts

```diff
@@ -45,15 +42,20 @@ export const render = async (req: Request, res: Response, next: NextFunction) =>
             } else {
                 return;
             }
-        }
+        } 
 
+        if (req.url == continueToPaymentPath) {
+                addAnotherDocumentUrl = `${BASKET_URL}`;
+                continueToPaymentUrl = `${await proceedToPayment(req, res, next)}`;
```

Comment:
What is the value of `continueToPaymentUrl` that results from this assignment?

Reply:
from the proceedToPayment method in the controler

Reply:
The other question I have about that particular assignment is why are you evaluating the `await proceedToPayment` in that manner (in a string expression)?